app/src/main/java/com/mithridat/nonoconverter/ui/editimage/EditImageActivity.java

```diff
@@ -325,7 +335,7 @@ public void onResume() {
         if (_columns == 0) {
             int bmWidth = _bmpCurrentImage.getWidth();
             int bmHeight = _bmpCurrentImage.getHeight();
-            _columns = bmWidth < 90 ? bmWidth : 45;
+            _columns = bmWidth < MAX_SIZE ? bmWidth : 45;
```

Comment:
Probably, it worth to replace "45" by constant (DEFAULT_SIZE, for example), how it was done with "15" and "90".